requirements.txt

```diff
@@ -1,6 +1,6 @@
 bottleneck
 cftime
-dask<=2021.02.0
+dask!=2021.3.0
```

Comment:
```suggestion
dask>=2021.3.0
```

Reply:
Both versions would work. Wouldn't they?

Reply:
probably. I prefer using <= that != for version numbers. In fact we can probably unpin. I think it's unlikely you get 2021.02.0